packages/icon/test/lion-icon.test.js

```diff
@@ -6,6 +6,47 @@ import hammerSvg from './hammer.svg.js';
 import '../lion-icon.js';
 
 describe('lion-icon', () => {
+  it('supports svg icon as a function which recieves a tag function as an argument and returns a tagged template literal', async () => {
+    const iconFunction = tag => tag`<svg id="svg"></svg>`;
```

Comment:
Maybe the test should give a better example than an id, because of potential a11y problems that arise (because icon is put in light dom)

Reply:
I don't mind, can you suggest smth concise and simple?

Reply:
maybe another attr like `data-identifier`

Reply:
done